src/packet/marker.js

```diff
@@ -32,31 +34,32 @@ import enums from '../enums';
  *
  * Such a packet MUST be ignored when received.
  * @memberof module:packet
- * @constructor
  */
-function Marker() {
-  this.tag = enums.packet.marker;
-}
+class MarkerPacket {
```

Comment:
Why rename class?

Reply:
All packet classes are renamed like this, the reason is so that you can do
```js
import { MarkerPacket } from 'openpgp';
```
and only import the packets you need, instead of having to do
```js
import { packet } from 'openpgp';
// do something with packet.Marker
```
This helps with tree shaking.